nvm.sh

```diff
@@ -781,7 +781,7 @@ nvm_strip_iojs_prefix() {
   if [ "${1-}" = "${NVM_IOJS_PREFIX}" ]; then
     nvm_echo
   else
-    nvm_echo "${1#"${NVM_IOJS_PREFIX}"-}"
+    nvm_echo "${1#${NVM_IOJS_PREFIX}-}"
```

Comment:
The nested quotes `"${NVM_IOJS_PREFIX}"` aren't necessary. And some editors will either show/highlight a syntax error or show treat the following as a big string, rather than valid bash (or sh) code.

![with-quotes](https://cloud.githubusercontent.com/assets/8132723/23636557/0d9cb19e-02fc-11e7-8efd-7f8194425f0a.png)

![without-quotes](https://cloud.githubusercontent.com/assets/8132723/23636556/0d9a06c4-02fc-11e7-9423-9b8814ffce88.png)